mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/GroupConsumerRetry.java

```diff
@@ -94,6 +94,10 @@ String asString() {
     Mono<Void> handleRetry(byte[] eventAsBytes, Event event, int currentRetryCount, Throwable throwable) {
         createStructuredLogger(event).log(logger -> logger.error("Exception happens when handling event after {} retries", currentRetryCount, throwable));
 
+        return handleRetry(eventAsBytes, event, currentRetryCount);
+    }
+
+    Mono<Void> handleRetry(byte[] eventAsBytes, Event event, int currentRetryCount) {
```

Comment:
are event and eventAsBytes the same event? In that case, you should not require the caller to pass both.